tests/modules/test_morpheus_modules.py

```diff
@@ -262,23 +262,21 @@ def test_file_loader_module():
     files = []
     file_types = ["csv", "parquet", "orc"]
     for ftype in file_types:
-        _tempfile = tempfile.NamedTemporaryFile(suffix=f".{ftype}", delete=False)
-        filename = _tempfile.name
+        with tempfile.NamedTemporaryFile(suffix=f".{ftype}", delete=False) as _tempfile:
```

Comment:
Use the pytest method for creating and working with temp files: https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html